cypress/support/a11y/checkAxe.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { context, defaultAxeConfig } from './defaultAxeConfig';
+
+function logViolations(violations) {
+  // Print any violations to the console using a custom callback
+  // https://github.com/component-driven/cypress-axe#using-the-violationcallback-argument
+  cy.task(
+    'log',
+    `${violations.length} violation${
+      violations.length === 1 ? '' : 's'
+    } ${violations.length === 1 ? 'was' : 'were'} detected.`
+  );
+
+  // Destructure keys from the violations object to create a readable table
+  const violationData = violations.map(({ id, description, impact, nodes }) => ({
+      id,
+      description,
+      impact,
+      nodes: nodes.length
+    })
+  );
+
+  // Print the table of violations to the console
+  cy.task('table', violationData);
+}
+
+Cypress.Commands.add('checkAxe', () => {
+  cy.injectAxe();
+  cy.checkA11y(context, defaultAxeConfig, logViolations);
+});
```

Comment:
It looks like this command is no longer accepting context or config overrides, unlike what the typescript definition suggests
```suggestion
Cypress.Commands.add('checkAxe', (context: ElementContext, config: RunConfig) => {
  cy.injectAxe();
  cy.checkA11y(context ?? defaultContext, config ?? defaultAxeConfig, logViolations);
});
```

Reply:
All other changes LGTM, I think this is my last code review comment!

Reply:
I added back the `context` and `config` arguments to the command call and tested the custom config object locally. Screenshot of a purposefully failing test attached. I also added typing to the custom violation callback function because YOLO! and I really like being able to trace data shapes.

---
### Deliberate failure (false positive)
<img width="1160" alt="Screen Shot 2022-02-07 at 11 41 47 AM" src="https://user-images.githubusercontent.com/934879/152849733-b58652c7-5b32-49e7-9078-542c566afba2.png">